From 051b912155115d0966d90917aa2b601d958461a4 Mon Sep 17 00:00:00 2001
From: Olliver Schinagl <oliver@schinagl.nl>
Date: Mon, 15 Apr 2019 09:49:06 +0200
Subject: [PATCH 2/3] unittests/libmtd_test: Include fcntl header

The test library for the mtd unit tests include various type's and
macro's that officially live in fcntl. Each file and header should
always properly include what they use, so lets add the fcntl headers.

Signed-off-by: Olliver Schinagl <oliver@schinagl.nl>
---
 tests/unittests/libmtd_test.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/unittests/libmtd_test.c b/tests/unittests/libmtd_test.c
index 633d94a..176d2ef 100644
--- a/tests/unittests/libmtd_test.c
+++ b/tests/unittests/libmtd_test.c
@@ -1,3 +1,4 @@
+#include <fcntl.h>
 #include <stdarg.h>
 #include <setjmp.h>
 #include <stddef.h>
-- 
2.20.1

